docs/api/paths/admin/db-_config-audit.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright 2022-Present Couchbase, Inc.
+#
+# Use of this software is governed by the Business Source License included
+# in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+# in that file, in accordance with the Business Source License, use of this
+# software will be governed by the Apache License, Version 2.0, included in
+# the file licenses/APL2.txt.
+get:
+  summary: Get database audit configuration
+  description: |-
+    Retrieve the audit configuration for the database specified.
+
+    Required Sync Gateway RBAC roles:
+
+    * TODO
```

Comment:
Do we have a ticket to decide RBAC roles?

Reply:
Following up in PRD